proof-of-spacetime.md

```diff
@@ -0,0 +1,197 @@
+## Proof-of-Spacetime
+
+This document describes Rational-PoSt, the Proof-of-Spacetime used in Filecoin.
+
+## Rational PoSt
+
+### Definitions
+
+| Name | Value |Description |
+|------|-------|------------|
+| `POST_PROVING_PERIOD` | `2880` blocks  (~24h) | The time interval in which a PoSt has to be submitted. |
+| `POST_CHALLENGE_TIME` | `240` blocks (~2h) | The time offset at which the actual work of generating the PoSt should be started. This is some delta before the end of the `Proving Period`, and as such less then a single `Proving Period`. |
```

Comment:
```suggestion
| `POST_CHALLENGE_TIME` | `240` blocks (~2h) | The time offset at which the actual work of generating the PoSt must not be started earlier than. This is some delta before the end of the `Proving Period`, and as such less then a single `Proving Period`. |
```

This phrasing might make the intent a little clearer, but also maybe not. just a suggestion

Reply:
maybe "can not"?